articles/dms/tutorial-mysql-azure-mysql-offline-portal.md

```diff
@@ -0,0 +1,254 @@
+
+2. On the **Select target** screen, specify the connection details for the target Azure Database for MySQL instance, and select **Next : Select databases>>**
+
+    ![Add target details screen](media/tutorial-mysql-to-azure-mysql-offline-portal/11-dms-portal-project-mysql-target.png)
+
+3. On the **Select databases** screen, map the source and the target database for migration, and select **Next : Configure migration settings>>**
+    
+    If the target database contains the same database name as the source database, Azure Database Migration Service selects the target database by default.
+    ![Select database details screen](media/tutorial-mysql-to-azure-mysql-offline-portal/12-dms-portal-project-mysql-selectdb.png)
+    
+    > [!NOTE] 
+    > Though you can select multiple databases in this step, each instance of Azure Database Migration Service supports up to 4 databases for concurrent migration. Also, there is a limit of 10 instances of Azure Database Migration Service per subscription per region. For example, if you have 80 databases to migrate, you can migrate 40 of them to the same region concurrently, but only if you have created 10 instances of the Azure Database Migration Service.
+
+4. On the **Configure migration settings** screen, select the tables to be part of migration, and select **Next : Summary>>**
+
+    ![Select tables screen](media/tutorial-mysql-to-azure-mysql-offline-portal/13-dms-portal-project-mysql-selecttbl.png)
+
+5. On the **Summary** screen, in the **Activity name** text box, specify a name for the migration activity and review the summary to ensure that the source and target details match what you previously specified.
+
+    ![Migration project summary](media/tutorial-mysql-to-azure-mysql-offline-portal/14-dms-portal-project-mysql-activitysummary.png)
+
+6. Select **Start migration**. The migration activity window appears, and the **Status** of the activity is **Initializing**. The **Status** changes to **Running** when the table migrations start.
+ 
+     ![Running migration](media/tutorial-mysql-to-azure-mysql-offline-portal/15-dms-portal-project-mysql-running.png)
+
+## Monitor the migration
+
+1. On the migration activity screen, select **Refresh** to update the display and see progress about number of tables completed. 
+
+2. You can click on the database name on the activity screen to see the status of each table as they are getting migrated. Select **Refresh** to update the display. 
+
+     ![Monitoring migration](media/tutorial-mysql-to-azure-mysql-offline-portal/16-dms-portal-project-mysql-monitor.png)
+
+## Complete the migration
+
+1. On the migration activity screen, select **Refresh** to update the display until the **Status** of the migration shows as **Complete**.
+
+    ![Complete migration](media/tutorial-mysql-to-azure-mysql-offline-portal/17-dms-portal-project-mysql-complete.png)
+
+## Perform migration cutover
```

Comment:
I am not using if we should call it "Perform migration cutover" as they aren't really doing a cutover. 
Maybe we should add the steps tp create logins, triggers etc as Post Migration Steps or something.

Reply:
Done